chart/values.schema.json

```diff
@@ -535,6 +535,10 @@
         "overwriteManifests": {
           "type": "string",
           "description": "OverwriteManifests can be used to overwrite the coredns manifests used to deploy coredns"
+        },
+        "priorityClassName": {
```

Comment:
I guess this also must find it's way into the docs: https://github.com/loft-sh/vcluster-docs/tree/main/docs/_partials/config/controlPlane/coredns

There is [`gen-partials.sh`](https://github.com/loft-sh/vcluster-docs/blob/main/hack/gen-partials.sh) which updates this. Might be that this done for the next beta release anyway though?! (cc @aimeeu)

Reply:
The process is manual and yes, we generate new partials for each release.

Reply:
so we should do it in another MR, right ?